test/es6/HTMLComments.js

```diff
@@ -22,7 +22,8 @@ WScript.Echo("Code before CRLF--> is reachable");
 --> WScript.Echo("Code after CRLF--> is unreachable");
 
 // CR
-WScript.Echo("Code before CR--> is reachable");
--> WScript.Echo("Code after CR--> is unreachable");
+WScript.Echo("Code before CR--> is reachable");
+--> WScript.Echo("Code after CR--> is unreachable");

```

Comment:
These carriage return (CR, `\r`, or `0x0d`) characters were deliberately placed here without newline (`\n` or `0x0a`) and are part of the parser test. Can you revert these changes?

Reply:
I don't recall editing that line possibly an oddity of my vscode settings when I made the change further down - will revert with a hex editor.

Reply:
Yeah this is one of those things that editors will generally try to make sane. Getting the file into this weird line ending state is almost a matter of just editing the bytes.